packages/bratus-app/src/components/ComponentCode/ComponentCode.jsx

```diff
@@ -0,0 +1,36 @@
+import { LoadingOutlined } from '@ant-design/icons';
+import { Divider } from 'antd';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import { vscDarkPlus } from 'react-syntax-highlighter/dist/esm/styles/prism';
+
+import { ComponentCodeRow } from './ComponentCode.sc';
+
+// nodeDetail gives us additional information about the component.
+const ComponentCode = ({ nodeDetail }) => {
+  if (!nodeDetail.visible) {
+    return <LoadingOutlined spin />;
+  }
+
+  return (
+    <div>
```

Comment:
You don't need this <div> because you already have the <ComponentNodeRow> parent wrapping the component